extensions/pointcloud/json-schema/schema.json

```diff
@@ -0,0 +1,130 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.stacspec.org/dev/extensions/pointcloud/json-schema/schema.json#",
+  "title": "Point Cloud Extension",
+  "description": "STAC Point Cloud Extension to a STAC Item",
+  "allOf": [
+    {
+      "$ref": "#/definitions/stac_extensions"
```

Comment:
Something went wrong here. You removed the STAC Item Ref instead of the stac_extensions Ref.